Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
```

Comment:
Using '1234' as a password for the database user is insecure. It's important to use a strong, unique password.